builder/builder-next/exporter/export.go

```diff
@@ -83,7 +83,7 @@ func (e *imageExporterInstance) Export(ctx context.Context, ref cache.ImmutableR
 	if ref != nil {
 		layersDone := oneOffProgress(ctx, "exporting layers")
 
-		if err := ref.Finalize(ctx); err != nil {
+		if err := ref.Finalize(ctx, true); err != nil {
```

Comment:
Not a super-fan of this boolean argument, but 🤷‍♂️

Reply:
That's in buildkit. Feel free to change it there, but I don't think it's a blocker for this PR.

Reply:
Yes, I saw it was implemented there :-)